src/main/java/domain/policy/TimePolicy.java

```diff
@@ -0,0 +1,17 @@
+package domain.policy;
+
+import java.time.LocalTime;
+
+public class TimePolicy {
+
+    private static final LocalTime OPERATING_START_TIME = LocalTime.of(8, 0);
+    private static final LocalTime OPERATING_END_TIME = LocalTime.of(23, 0);
+
+    public void validateOperatingTime(LocalTime time) {
+        if(time.isBefore(OPERATING_START_TIME)
```

Comment:
코드 전체적으로 [Java Google Style Guide](https://github.com/woowacourse/woowacourse-docs/tree/main/styleguide/java)를 잘 적용했는지 확인해볼까요?
띄워쓰기 같은 경우는 IntelliJ의 도움을 받으면 보다 손쉽게 적용할 수 있습니다. [참고](https://eroul-ri.tistory.com/26)
```suggestion
        if (time.isBefore(OPERATING_START_TIME)
```